config/initializers/convert_time_to_eo.rb

```diff
@@ -0,0 +1,12 @@
+# temporary hack for when/if we can get patch accepted upscream
+# et-orbi doesn't support comparisons of EoTime in the scheduler with user provided Time objects
+# see https://github.com/ManageIQ/manageiq/pull/19153#issuecomment-521719495
+module ConvertTimeToEoTime
+  def ==(*args)
+    o = EtOrbi.make_time(args.first) if args.first.kind_of?(Time)
```

Comment:
Do we need to also support DateTime type?

Reply:
I don't think so but it won't hurt.